<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterEditor</class>
 <widget class="QWidget" name="FilterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QComboBox" name="typeCombo"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="valueEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="keyEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="keyLabel">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
